src/extension.ts

```diff
@@ -49,13 +52,21 @@ async function init(context: vscode.ExtensionContext, git: ApiImpl, repository:
 	context.subscriptions.push(vscode.window.registerUriHandler(uriHandler));
 	context.subscriptions.push(new FileTypeDecorationProvider());
 
-	const prManager = new PullRequestManager(repository, telemetry, git);
+	const credentialStore = new CredentialStore(telemetry);
+
+	const prManager = new PullRequestManager(repository, telemetry, git, credentialStore);
 	context.subscriptions.push(prManager);
 
 	const reviewManager = new ReviewManager(context, repository, prManager, tree, telemetry);
 	tree.initialize(prManager);
 	registerCommands(context, prManager, reviewManager, telemetry);
 
+	const gitExtension = vscode.extensions.getExtension<GitExtension>('vscode.git')!.exports;
+	const gitAPI = gitExtension.getAPI(1);
```

Comment:
@alexr00 I've extracted getting the `GitAPI` here, so it can be reused in initializing `GithubRemoteSourceProvider`.